doc/man1/openssl-kdf.pod.in

```diff
@@ -66,7 +66,7 @@ cases.
 =item B<-kdfopt> I<nm>:I<v>
 
 Passes options to the KDF algorithm.
-A comprehensive list of parameters can be found in the EVP_KDF_CTX
```

Comment:
Maybe reword to
Refer to L<EVP_KDF(3)/PARAMETERS> for a list of parameters.

Reply:
done